dev/customer_testing/lib/customer_test.dart

```diff
@@ -73,9 +73,6 @@ class CustomerTest {
     if (!fetch[1].contains(_fetch2)) {
       throw FormatException('${errorPrefix}Second "fetch" directive does not match expected pattern (expected "git -C tests checkout HASH").');
     }
-    if (update.isEmpty) {
```

Comment:
To make sure this is really unused, could this PR also not add to `update` on line 28?

Reply:
Done